### PR TITLE
Fixes error 500 issue on some test schemas

### LINKED ISSUE #1074: Error 500 on several test schemas
### Expected behaviour
Must navigate to confirmation/summary screen

### Actual behaviour

Giving an Error 500 as some of this test schemas does not have a summary block

### Steps to reproduce the behaviour
Launch test_textfield schema
enter data in all fields click [Save and Continue]

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?
Fixes the error 500 issue when the test schemas do not have submission block. fixes #1074

### How to review 
Retest defect #1074

### CODE REVIEW

**Comment by collisdigital** [comment]
You need to run the linting locally and fix the errors:

`./scripts/run_lint.sh`

This is what Travis is failing on above https://travis-ci.org/ONSdigital/eq-survey-runner/builds/216815837

File: tests/app/data/test_schema_validation.py [hunk 1]
```diff
@@ -83,6 +84,19 @@ def test_no_duplicate_ids_in_schema(self):
...
+        schema_files = self.all_schema_files()
+
+        for schema_file in schema_files:
+            confirmation_or_summary_in_block = False
+            with open(schema_file, encoding="utf8") as file:
+                schema_json = load(file)
+                blocks = SchemaHelper.get_blocks(schema_json)
+                for block in blocks:
+                    if block['type'] in ["Summary", "Confirmation"]:
+                        confirmation_or_summary_in_block = True
```
Comment (collisdigital) [inline tests/app/data/test_schema_validation.py]: You can `break` after this, there's no need to keep looping once you've found one.

**Review by collisdigital — APPROVED** [review]
Just need to fix the linting...

File: tests/app/data/test_schema_validation.py [hunk 2]
```diff
@@ -83,6 +83,19 @@ def test_no_duplicate_ids_in_schema(self):
...
+            self.assertTrue(confirmation_or_summary_in_block, msg='{file} does not has a confirmation or summary page'.format(file=schema_file))
```
Comment (jonnyshaw89) [inline tests/app/data/test_schema_validation.py]: Typo `does not **have** a confirmation or summary page`

File: tests/app/data/test_schema_validation.py (hunk 1 above)
Comment (jonnyshaw89) [reply tests/app/data/test_schema_validation.py]: +1 this would be good practice and should be done.
However it should be the last block anyway.

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: tests/app/data/test_schema_validation.py (hunk 1 above)
Comment (collisdigital) [reply tests/app/data/test_schema_validation.py]: Good point, maybe this should loop backwards through the blocks!

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
@yoganandkunche squash, rebase?

**Pull request merged**